testing/endtoend/helpers/helpers.go

```diff
@@ -33,6 +33,7 @@ const (
 	cpuProfileFileName  = "node_cpu_profile_%d.pb.gz"
 	fileBufferSize      = 64 * 1024
 	maxFileBufferSize   = 1024 * 1024
+	AltairE2EForkEpoch  = 6
```

Comment:
Seems out of the scope for this PR but can't hurt, Please update the PR description though!